src/main/java/transportservice/RunPlugin.java

```diff
@@ -17,21 +20,23 @@
 import org.opensearch.transport.*;
 import transportservice.netty4.Netty4Transport;
 
-import java.net.*;
+import java.io.IOException;
 import java.util.Collections;
 import java.util.List;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 import static java.util.Collections.emptySet;
+import static org.opensearch.common.UUIDs.randomBase64UUID;
 
 public class RunPlugin {
 
+    public static final String REQUEST_EXTENSION_ACTION_NAME = "internal:discovery/extensions";
     private static final Settings settings = Settings.builder()
-        .put("node.name", "NettySizeHeaderFrameDecoderTests")
+        .put("node.name", "node_extension")
         .put(TransportSettings.BIND_HOST.getKey(), "127.0.0.1")
-        .put(TransportSettings.PORT.getKey(), "9301")
+        .put(TransportSettings.PORT.getKey(), "4532")
```

Comment:
Lets get these properties from `extension.yml`. 
I know you have a PR, just link the issue :)

Reply:
Addressed in https://github.com/owaiskazi19/IndependentPlugin/pull/10